qiita_ware/test/test_commands.py

```diff
@@ -144,14 +145,12 @@ def test_submit_EBI_parse_EBI_reply_failure(self):
         with self.assertRaises(ComputeError):
             submit_EBI(ppd.id, 'VALIDATE', True)
 
+    @skipIf(environ.get('ASPERA_SCP_PASS', ''), 'skip: ascp not configured')
```

Comment:
This means that the test will be skipped if the password is present. I think it would have to be:

```
@skipIf(environ.get('ASPERA_SCP_PASS', '') is None, 'skip: ascp not configured')
```

Reply:
Fixing, thanks!